src/System.Net.Sockets/tests/FunctionalTests/UnixDomainSocketTest.cs

```diff
@@ -348,6 +354,21 @@ public void ConcurrentSendReceiveAsync()
             }
         }
 
+        [Fact]
+        [PlatformSpecific(TestPlatforms.AnyUnix)]  // Tests new UnixDomainSocketEndPoint throws the correct exception for invalid args
+        public void CreateWithInvalidPathThrows()
```

Comment:
Test names (methods) should follow the standard convention:

""CreateWithInvalidPathThrows()" -> UnixDomainSocketEndPoint_InvalidPaths_Throws()

Reply:
Fixed in the latest change.